tests/fixed/constants-var.php

```diff
@@ -6,7 +6,13 @@
 
 const FOO = 123;
 
+const BAR_1 = 1;
+const BAR_1 = 2;
+
 class Bar
 {
     public const BAZ = 456;
+
+    protected const PROPERTY_1 = '1';
+    protected const PROPERTY_2 = '2';
```

Comment:
Spacing between constants will be proposed in a separate PR.

Spoiler: [SlevomatCodingStandard.Classes.ConstantSpacing](https://github.com/slevomat/coding-standard#slevomatcodingstandardclassesconstantspacing-).